### DISCUSSION TITLE
[Feature] Retain captions/description information from iPhones when backing up

### DISCUSSION THREAD

**Discussion by fivestones** [immich-app/immich#4220] [Feature Request]
### The feature

Immich should capture the caption or description of a photo when backing up from iPhones, and this should be placed in the description in Immich for that photo.

Right now as far as I can see, this caption information doesn't get picked up when backups are made. If I have a photo with a caption, then use the iPhone immich app to send this to my Immich server, when I view that photo in Immich, the description field is empty.

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by bobcqld53** [comment]
Agree  , I have thousands of holiday pics with captions/descriptions ...

**Comment by tlvince** [comment]
Added in https://github.com/immich-app/immich/pull/6344. Should land in the next release.

**Reply by fivestones** [reply]
Awesome. Thanks!!

**Reply by MotasR** [reply]
Sorry to bump this after more than a year, but this doesn't seem to work anymore. Could you confirm this or would it be better to create a separate issue?

iOS 18.5
App version: 1.132.3 build 205
Server version: 1.132.3

**Reply by tlvince** [reply]
Please open a new issue.

**Reply by blue-kaleidoscope** [reply]
@MotasR Was it ever reported as an issue? Looks like someone opened a feature request for it: #19798
